packages/linux/package.mk

```diff
@@ -110,6 +110,11 @@ post_patch() {
       sed -e "s|^CONFIG_DRM_LIMA=.*$|# CONFIG_DRM_LIMA is not set|" -i $PKG_BUILD/.config
       sed -e "s|^CONFIG_DRM_PANFROST=.*$|# CONFIG_DRM_PANFROST is not set|" -i $PKG_BUILD/.config
     fi
+
+    # disable wireguard support if not enabled
+    if [ ! "$WIREGUARD_SUPPORT" = yes ]; then
```

Comment:
```suggestion
    if [ "$WIREGUARD_SUPPORT" != "yes" ]; then
```

Reply:
This is how i'd normally do it, but I was following prior art (read above). Ignore or change them all?

Reply:
Eugh. No you're correct. If you don't mind changing them all that would be nice, otherwise leave as is and maybe one day we'll fix them all up! :)

Reply:
I leave as-is for now.